dbms/src/Common/MemoryTracker.cpp

```diff
@@ -226,6 +226,11 @@ void submitLocalDeltaMemory()
     local_delta = 0;
 }
 
+Int64 getLocalDeltaMemory()
+{
+    return local_delta;
```

Comment:
`local_delta` is thread-local, so different threads will see different deltas, is that what you expect?

Reply:
https://github.com/pingcap/tiflash/pull/4570/files#diff-773c77885e15377996db622e99646f024c4815893a7d44878db9bc05bbc4bf6fR612

Reply:
Yes